README.md

```diff
@@ -155,6 +155,20 @@ install-nix-action's own `extra_nix_config` input:
       substituters = https://hydra.iohk.io https://cache.nixos.org/
 ```
 
+### How do I use `nix develop`?
+
+`nix develop` can be used for `steps[*].shell`.
+
+```yaml
+  # optional step: build devShell in advance for accuracy of subsequent step timing and result
```

Comment:
> for accuracy of subsequent step timing and result

I'm not sure what "result" is referring to here. I'm guessing the point is to be able to tell that the shell build failed vs the run command?

I would say "pre-build the shell separately to avoid skewing the run time of the next step and have clear point of failure should the shell fail to build".

N.B. there's a chance that the shell could be subsequently GC'd, but that's out of scope for us.